src/debug.c

```diff
@@ -175,8 +158,8 @@ dump_tree(struct server *server, struct wlr_scene_node *node,
 	printf("%s %*c %4d  %4d  [%p]\n", type, padding, ' ', x, y, node);
 
 	if ((IGNORE_MENU && node == &server->menu_tree->node)
-			|| (IGNORE_SSD && view && view->ssd.tree
-			&& node == &view->ssd.tree->node)) {
+			|| (IGNORE_SSD && view
+			&& node == ssd_get_tree_node(view->ssd))) {
```

Comment:
Maybe this could be `ssd_is_treenode(node)` instead?

Reply:
Changed to `ssd_debug_is_root_node()`